src/visualizers-workbench/P5VisualizerTemplate.ts

```diff
@@ -0,0 +1,183 @@
+import p5 from 'p5' // we need the p5.Color type to declare top-level colors
+import {P5Visualizer} from '../visualizers/P5Visualizer'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+
+/** md
+# Entries (p5 Template)
+
+[<img
+  src="../../assets/img/Entries/1729.png"
+  width=700
```

Comment:
This and the new image for Differences look _huge_ on my screen. Are you sure you want width=700? Consider dialing that down a bit...

Reply:
In fact, leafing through them, to my eye the most attractive doc page is the one for Factor Histogram that has the example image floated right. Do we want to emulate that for the new template visualizer and Differences images, to start a trend/convention?

Reply:
Here's what I see in Firefox 125.0 on Ubuntu 22.04. When the window gets too narrow, the image scales down to the width of the text column, preserving its aspect ratio. If you're seeing something different, could you show me? It might also be helpful to look at the Grid visualizer page. I set the Entries thumbnail to display at the same height as the Grid thumbnail: 320 px. If Entries looks huge, but Grid doesn't, something weird is going on.

![image](https://github.com/numberscope/frontscope/assets/12519587/51df3407-a6a9-4a1b-89de-51d3d6310581)

Reply:
Yeah, no, that's exactly the sort of thing I see. It just looks overkill enormous to me, graphic-design wise. And you can only see a trifle of the text until you scroll, especially on a wide-screen monitor.  I don't think the analogy with Grid is entirely apt, because the aspect ratios of the images are so different. "Same pixel height" is not perceptually the same thing as "same size".  Personally, if I were to advocate for a standard doc page styling here, I'd float all the initial example images right at something like 50%-40% of the width (maybe the exact percentage would vary depending on the aspect ratio). That way, you see both the image and the introductory text, which to me is very inviting. The only drawback I can think of off the top of my head to that is I don't think it's the right layout for mobile in portrait mode. There I think the better layout is centered, non-floated, at about 30-40% of the visible height. Again, you see the image and a decent amount of the intro text below it. 

Perhaps we should create a CSS class available in the doc pages of say `featured-image` and set it up with media properties (or whatever they are) so that it is float-right at 45% width down to (say) a 0.8 aspect ratio and then switches to centered at 35% viewport height for lower aspect ratios, and then use that class on these new images and encourage folks to use it on future leading images for visualizers? (I mean, if that graphic design sounds good to you...)

Reply:
OK, we are leaving this to a "Uniformize graphic design/presentation of Visualizer docs" issue for the future. Resolving here.